test/python/transpiler/test_dependency_graph.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests the DependencyGraph."""
+import unittest
+
+from qiskit import ClassicalRegister, QuantumRegister, QuantumCircuit
+from qiskit.test import QiskitTestCase
+from qiskit.transpiler.passes.mapping.algorithm.dependency_graph import DependencyGraph
+
+
+class TestDependencyGraph(QiskitTestCase):
+    """Tests the DependencyGraph."""
+
+    def test_qargs(self):
+        """ Test for qargs.
+        """
+        qr = QuantumRegister(2, 'q')
+        cr = ClassicalRegister(2, 'c')
+        circuit = QuantumCircuit(qr, cr)
+        circuit.cx(qr[1], qr[0])
+        circuit.measure(qr[0], cr[1])
+        dep_graph = DependencyGraph(circuit)
+        self.assertEqual(dep_graph.qargs(0), [(qr, 1), (qr, 0)])
```

Comment:
```suggestion
        self.assertEqual(dep_graph.qargs(0), [qr[1], qr[0]])
```
We are not using tuples anymore to refer to qubits.